pkg/database/database.go

```diff
@@ -215,7 +219,10 @@ func OpenDB(dbName string, multidbHandler sql.MultiDBHandler, op *Options, log l
 		return nil, err
 	}
 
-	dbi.Logger.Infof("SQL Engine ready for database '%s' {replica = %v}", dbName, op.replica)
+	dbi.documentEngine, err = document.NewEngine(dbi.st, sql.DefaultOptions().WithPrefix([]byte{DocumentPrefix}))
```

Comment:
it should not be related to sql, at least externally

`sql.DefaultOptions()`-> `document.DefaultOptions()`